docs/source/install.rst

```diff
@@ -122,7 +122,7 @@ of the script installed with this package::
 
     export WORKON_HOME=$HOME/.virtualenvs
     export PROJECT_HOME=$HOME/Devel
-    source /usr/local/bin/virtualenvwrapper.sh
+    source ~/.local/bin/virtualenvwrapper.sh
```

Comment:
The `/usr/local` prefix is not ideal, but if this path only applies on Ubuntu then it's not ideal either. Maybe we need to clarify that the path to use here is going to depend on how virtualenvwrapper is installed?